### PR TITLE
Bug-Fix: Handle unknown routes or 404 not found page

### LINKED ISSUE #192: [bug]: Unknown routes are not handled or not "not-found" page
### Describe the bug

If a user mistakenly navigates to a unknown route, then there is no option to navigate back to home, its just showing a blank page 

### To Reproduce

Go to 'https://wemakedevs.org'
Click on 'search tab or route tab'
Add 'a random name after slash for instance `.../unknown`

### Expected Behavior

A blank page will pop up instead a `not-found` page is expected which navigates back a user to home page

### Screenshot/ Video

Screenshot from 2022-12-11 12-40-01

### Needs
- It will enhance user experience
- It is better to notify a user if something goes wrong so that user can proceed with clarity

### PR DESCRIPTION
**Pull request by sboy99**
## Fixes Issue

This PR fixes the following issues:
If a user navigates to a unknown routes an `Error Page` will pop up now instead of a blank page. In the error page there will be a `Back to Home ` button, by clicking it user will navigate to  `Home Page` 

closes #192 

## Changes proposed
- Added a route with `*` path which will automatically handle unknown/random routes that are now listed.
- Added a Error Page that will pop up on unknown route
- Added some CSS to the Error Page 
- Added functionality to the `Back to Home` button
Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarised content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots 
After:
Screenshot from 2022-12-11 15-51-57
Add all the screenshots which support your changes

### CODE REVIEW

**Review by Aadarsh805 — APPROVED** [review]
yep good work

**Comment by sboy99** [comment]
Thanks ☺️

File: src/v2/pages/errorPage/index.module.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  color: white;
+  text-transform: capitalize;
+}
+.errorPageSection .container {
+  padding: 1rem;
+  width: 100%;
+  max-width: 30rem;
+}
+.errorPageSection .container .highHeading {
+  font-size: 4rem !important;
```
Comment (FrozenHearth) [inline src/v2/pages/errorPage/index.module.css:14]: Can we avoid using !important here? Rather, increase the specificity.

Comment (sboy99) [reply src/v2/pages/errorPage/index.module.css:14]: As this is a individual page, for ui improvement i set it to ! important.
It was inheriting a large font size from its ancestors that was not improving ui experience as it was making the font so big. As the ui design was not pre-instructed I did so and so.
However, if any changes that would match organizations' defaults, I'm ready to make changes, Thank you!

**Comment by Chandraprakash-Darji** [comment]
@sboy99 Code base has been restructured so just move your file to the appropriate path.

**Comment by sboy99** [comment]
Hey, @Chandraprakash-Darji  I have created a new pull request regarding the changes. Also created a new branch that has no conflict with base branch. Please review it #231 
Thank You

**Pull request closed without merging**